tests/css_parens/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -22,4 +37,18 @@ a {
   );
 }
 
+.foo {
+  --paddingC: calc(var(--widthB) / 2);
+  content: attr(data-title);
+  color: var(--main-bg-color);
+  background-color: rgb(255, 0, 0);
+  background: element(#css-source);
+  width: calc(100% - (#{var(--g-spacing)} - #{$iframe-x-padding}) * 2);
+  padding-bottom: (
+    100% * $image-height / ($image-width-responsive + $image-margin-responsive *
+          2)
```

Comment:
This doesn’t look very pretty. Would you be willing to fix this in this PR, or should I open a new issue?

Reply:
@j-f1 in this PR just tests, to ensure all extra spaces before and after parens and signs was removed. We already have issue https://github.com/prettier/prettier/issues/1811